topics/assembly/tutorials/unicycler-assembly/tutorial.md

```diff
@@ -0,0 +1,272 @@
+
+## Annotation with Prokka
+
+
+> ### {% icon hands_on %} Hands-on: Annotation
+> 
+> 1. **Prokka** {% icon tool %} with the following parameters:
+>   - "Contigs to annotate" parameter to the assembly ouput of `Unicycler`
+>   - "Genus name" parameter to `Escherichia`
+>   - "Species name" parameter to `coli`
+>   - "Strain name" parameter to `C-1`
+>   - "Use genus" parameter to `yes`
+> 
+{: .hands_on}
+
+Prokka outputs 10 datasets (including two log files). These are in various formats: 
+
+* **txt** : Provides Statistics on the annotation : number of CDS predicted, number of rRNA etc.
+* **tbl** : Provides a tabulated list of annotated features.
+* **fsa** : Nucleotide fasta file of the input contig sequence.
+* **sqn** : [ASN.1](https://www.ncbi.nlm.nih.gov/Structure/asn1.html) format file for submission to GenBank.
+* **ffn** : Nucleotide FASTA file of all the prediction transcripts.
+* **faa** : Protein FASTA file of the translated CDS sequences.
+* **fna** : Nucleotide fasta file of the input contig sequence.
+* **gbk** : [GenBank](https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html) file.
+* **gff** : [gff3](http://gmod.org/wiki/GFF3) file.
+
+## Visualize the results in IGV
+
+Let's look at the entire assembly and its annotation in the genome browser. We can do this using Integrated Genome Browser ([IGV](http://software.broadinstitute.org/software/igv/)). 
+
+Visualization requires a local installation of IGV. If you have IGV installed - just start it. If you don't - read on. 
+
+#### Starting IGV
+
+Go to IGV [download page](http://software.broadinstitute.org/software/igv/download) and select one of the options. The one I would try first would be **Jave Web Start**.Simply click the **Launch** button for 10 GB distribution. 
+
+
+> ### {% icon hands_on %} Hands-on: Visualization in IGV
+> 
+> 
+> 1. Start IGV, it will look something like this:
+> 
+>      ![IGV just started](../../images/igv1.png "IGV started by using Java Web Start. Note that it is currently showing Human Genome (hg38) build. This is obviously <b>not</b> what we want.")
+> 
+> 2. Locate the output of Unicycler and expand it :
+> 
+>      ![Unicycler result Visualization](../../images/unicycler_igv.png  "Expanding Unicycler assembly reveals IGV link.")
+> 
+> 3. Click on the *local* link highlighted with orange outline. The browser will change:
+> 
+>      ![IGV with Unicycler assembly](../../images/igv2.png "Now you can see the major contig shown in the browser window.")
+> 
+> 4. Let's add Prokka annotations to the browser image. For this simply expand Prokka's GFF3 dataset and click on the *local* link:
+> 
+>      ![Expanded GFF3 dataset representing Prokka annotations](../../images/prokka_item.png "Expanded GFF dataset generated with Prokka. Click on the <em>local</em> link (highlighted with orange outline) to display this dataset within IGV.")ßßß
```

Comment:
Are the `ßßß` normal or did you add them by mistake?

Reply:
It was a mistake, I removed it. Thanks for seing it !